packages/vue-form/src/components/Password.vue

```diff
@@ -75,9 +79,18 @@ const props = defineProps({
 
 defineEmits(["update:modelValue"]);
 
+const translatedErrorMessages = {
+  required: props.errorMessages.required
+    ? props.errorMessages.required
+    : t("validation.errors.password.required"),
+  weak: props.errorMessages.weak
+    ? props.errorMessages.weak
+    : t("validation.errors.password.weak"),
+};
```

Comment:
this is necessary as defineProps is a runtime macro, and we can not use any local variable inside it.